client/state/analytics/actions.js

```diff
@@ -24,7 +24,7 @@ const joinAnalytics = ( analytics, action ) =>
 export const composeAnalytics = ( ...analytics ) => ( {
 	type: ANALYTICS_MULTI_TRACK,
 	meta: {
-		analytics: analytics.map( property( 'meta.analytics' ) )
+		analytics: analytics.map( property( 'meta.analytics[0]' ) )
```

Comment:
If you find a unique case that failed for you in code, it's usually helpful to add a test that fails, and then goes green with your updated code. This is commonly known as red, green, refactor used in TDD. 

Since the test is green in either case, we can argue that it's either not good enough or we failed to capture a case we care about.